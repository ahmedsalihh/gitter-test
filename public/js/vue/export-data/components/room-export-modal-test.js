const mount = require('../../__test__/vuex-mount');
const { default: RoomExportModal } = require('./room-export-modal.vue');

describe('Room export modal', () => {
  it('matches snapshot', () => {
    const { wrapper } = mount(RoomExportModal, {
      roomId: '123'
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
