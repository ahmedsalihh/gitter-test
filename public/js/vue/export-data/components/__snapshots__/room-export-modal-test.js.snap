// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Room export modal matches snapshot 1`] = `
<export-data-modal-stub
  title="Export room data"
>
  <p
    class="export-rate-limit-note"
  >
    
    Each export link is rate-limited to once every 3 hours. You will see a "429 - Too many
    requests" error once you reach the limit.
  
  </p>
   
  <ul
    class="export-link-list"
  >
    <li>
      <a
        href="/api_web/export/rooms/123/messages.ndjson"
        target="_blank"
      >
        Messages
      </a>
    </li>
     
    <li>
      <a
        href="/api_web/export/rooms/123/users.ndjson"
        target="_blank"
      >
        Users
      </a>
    </li>
     
    <li>
      <a
        href="/api_web/export/rooms/123/integration-events.ndjson"
        target="_blank"
      >
        Integration events
      </a>
    </li>
     
    <li>
      <a
        href="/api_web/export/rooms/123/bans.ndjson"
        target="_blank"
      >
        Bans
      </a>
    </li>
  </ul>
</export-data-modal-stub>
`;
